src/ng-multiselect-dropdown/test/multi-select.component.spec.ts

```diff
@@ -12,13 +12,13 @@ class Ng2MultiSelectDropdownSingleSelect {
   @ViewChild(MultiSelectComponent)
   select: MultiSelectComponent;
   cities = [
-    { item_id: 1, item_text: 'Mumbai' },
-    { item_id: 2, item_text: 'Bangalore' },
-    { item_id: 3, item_text: 'Pune' },
-    { item_id: 4, item_text: 'Navsari' },
-    { item_id: 5, item_text: 'New Delhi' }
+    { item_id: 1, item_text: 'Mumbai', isDisabled: false },
```

Comment:
Please undo these changes. It should work without isDisabled property as it is optional now.

Reply:
You may have one item with isDisabled true,and have respective test case.

Reply:
Done.